src/S3ClientGUI/s3treemodel.cpp

```diff
@@ -106,6 +123,28 @@ void S3TreeModel::setRootPath(const QString &path) {
     qDeleteAll(m_currentData);
     m_currentData.clear();
 
+	//loading
+	QVariantList loadData;
+	loadData << "...loading [-]" << QVariant() << QVariant() << QVariant();
+	m_currentData.append(new SimpleItem(loadData, S3LoadingType, "", ""));
+
+	QObject::connect(timer, &QTimer::timeout, this, [=]()
+	{
+		if (m_currentData.size() > 0)
```

Comment:
from line 133 ~ 139,  what for ?

Reply:
for just out "loading". there ware out  ".." and "loading" at the same time

Reply:
I try again,it's ok to delete.